src/entityCreation/README.md

```diff
@@ -0,0 +1,141 @@
+# 🧱 Entity Management - Creation Flow
+
+This guide provides step-by-step instructions to manage entities in the SAAS platform across different roles and environments.
+
+---
+
+## 🔐 Auth Keys & Tokens
+
+### 🔸 Org Admin - QA
+
+| Key                     | Value                     |
+| ----------------------- | ------------------------- |
+| `internal-access-token` | `{internal-access-token}` |
+| `X-auth-token`          | `{Token}`                 |
+
+### 🔸 Super Admin - QA
+
+| Key                     | Value                     |
+| ----------------------- | ------------------------- |
+| `internal-access-token` | `{internal-access-token}` |
+| `X-auth-token`          | `{Token}`                 |
+| `admin-auth-token`      | `{admin-auth-token}`      |
+| `tenantId`              | `shikshagraha`            |
+| `orgId`                 | `blr`                     |
+
+---
+
+## 🌐 Origin URLs (Per Organization)
+
+| Organization | Origin URL                      |
+| ------------ | ------------------------------- |
+| Shikshalokam | `shikshalokam-qa.tekdinext.com` |
+| Shikshagraha | `shikshagrah-qa.tekdinext.com`  |
+| Super Admin  | `default-qa.tekdinext.com`      |
+
+---
+
+## 🔑 Login API
+
+<details>
+<summary>Login API</summary>
+
+```bash
+curl --location 'https://saas-qa.tekdinext.com/user/v1/account/login' \
+--header 'Content-Type: application/json' \
+--header 'origin: shikshalokam-qa.tekdinext.com' \
+--data-raw '{
+  "identifier": "orgadmin@sl.com",
```

Comment:
@MallanagoudaB we dont need to expose the credentials, 

identifier : 'email/phone'
password: 'password'